includes/rest-api/class-sensei-rest-api-internal.php

```diff
@@ -51,6 +51,7 @@ public function register() {
 			new Sensei_REST_API_Lesson_Quiz_Controller( $this->namespace ),
 			new Sensei_REST_API_Question_Options_Controller( $this->namespace ),
 			new Sensei_REST_API_Extensions_Controller( $this->namespace ),
+			new Sensei_REST_API_Messages_Routes_Controller( $this->namespace ),
```

Comment:
Not sure about the naming here. Could not come up with something better. The `Sensei_REST_API_Messages_Controller` is already taken, but that is also not used. Not sure if it could be removed or touched.

Reply:
I think it can be part of `Sensei_REST_API_Messages_Controller`, it's for the same feature. If we want to break it up, maybe something like `...Send_Messages_Controller`?

Reply:
Renamed to `Sensei_REST_API_Send_Message_Controller`. b571076ee61b0aae1d637b6e5e1e432fab5850dc